171537499 health viewer

Adrian-de-Lucas-Gomez opened the pull request:
Ya está hecho el HealthViewer , completamente funcional y también se ha añadido a ComponentType y las imágenes correspondientes a vida activa o desactivada en resources para tener rápido acceso.

Adrian-de-Lucas-Gomez committed:
Comienzo de HealthViewer

Toda la parte de codigo está hecha. Se ha puesto un metodo en Health para que podamos saber la vida del jugador y mostrarla correctamente.

Adrian-de-Lucas-Gomez committed:
Imagenes de la vida y bajado el volumen

Para que tengamos imagenes de testeo y he bajado el volumen que estaba muy alto

Adrian-de-Lucas-Gomez committed:
Comprobacion de que funciona y se dibuja correctamente

Se ha comprobado que el componente HealthViewer muestra las vidas disponibles y perdidas que tiene el jugador. Se le pasa un parámetro para colocarlo donde nos interese de la pantalla según para que jugador sea.

Adrian-de-Lucas-Gomez committed:
Merge branch 'master' into 171537499-HealthViewer

Adrian-de-Lucas-Gomez committed:
Ajustes tontos

No tenia puesto en el draw que fuera constante, solo eso.

JuanmaSalva committed:
el tinkie ya tiene la vida asociada

Si haces que pierda una vida, la barra de vida correspondiente pasará a ser de color gris

JuanmaSalva reviewed (approved):
gg

EstebenR reviewed (changes requested):
Que se le pase el enum en el creador y luego en el init lo pide, (ver componente viewer)

JuanmaSalva reviewed (changes requested):
Las barras tienen que ser objetos de box2D pero estáticos y triggers

Adrian-de-Lucas-Gomez committed:
Solucionadas las peticiones

Se ha puesto que se pasa la ID de la textura en la constructora y se busca en la propia clase. Tambien se ha aclarado el tema de si era fisico o no, llegando a la conclusión de que no es necesario pero si se va a usar el vertor de Box2D.

EstebenR reviewed (approved):
nice

JuanmaSalva reviewed (approved):
yep
